README.md

```diff
@@ -1,23 +1,20 @@
-[![Build
-Status](https://travis-ci.org/vitormattos/blueprint-sdk-maker.svg?branch=master)](https://travis-ci.org/vitormattos/blueprint-sdk-maker)
-[![Coverage
-Status](https://coveralls.io/repos/vitormattos/blueprint-sdk-maker/badge.svg?branch=master&service=github)](https://coveralls.io/github/vitormattos/blueprint-sdk-maker?branch=master)
-[![Latest Stable
-Version](https://poser.pugx.org/vitormattos/blueprint-sdk-maker/v/stable)](https://packagist.org/packages/vitormattos/blueprint-sdk-maker)
-[![License](https://poser.pugx.org/vitormattos/blueprint-sdk-maker/license)](https://packagist.org/packages/vitormattos/blueprint-sdk-maker)
+[![Build Status](https://travis-ci.org/vitormattos/blueprint-sdk-maker.svg?branch=master)](https://travis-ci.org/vitormattos/blueprint-sdk-maker)
+[![Coverage Status](https://coveralls.io/repos/vitormattos/blueprint-sdk-maker/badge.svg?branch=master&service=github)](https://coveralls.io/github/vitormattos/blueprint-sdk-maker?branch=master)
 [![PHPStan](https://img.shields.io/badge/PHPStan-enabled-brightgreen.svg?style=flat)](https://github.com/phpstan/phpstan)
+[![Latest Stable Version](https://poser.pugx.org/vitormattos/blueprint-sdk-maker/v/stable)](https://packagist.org/packages/vitormattos/blueprint-sdk-maker)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207-blue.svg)](https://php.net/)
+[![License](https://poser.pugx.org/vitormattos/blueprint-sdk-maker/license)](https://packagist.org/packages/vitormattos/blueprint-sdk-maker)
 
-# Parser for API blueprint files
+# API Blueprint Parser
 
-[API Blueprint](https://apiblueprint.org/) is a powerful high-level API
-description language for web APIs and generate `.apib` media files. With this
-project you will parse `.apib` files and get all propertyes of file with
-oriented object interface.
+> [API Blueprint](https://apiblueprint.org/) is a powerful high-level API description language for web APIs. 
 
+Through that's project it's possible parse `.apib` files, get all their properties and generate all files needed to push a new SDK automatically.
```

Comment:
The files generated to new sdk don't is automatically pushed to new repository. Believe exist another word for express this, another possible solution is remove the word `automatically`